conf/surveys/CFHT.yaml

```diff
@@ -1,28 +1,14 @@
-# http://www.cfht.hawaii.edu/Instruments/Imaging/Megacam/generalinformation.html
-# http://www.cfht.hawaii.edu/Instruments/ObservatoryManual/om-focplndat.gif
-# Calculating zeropoints with:
-# http://www1.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/community/CFHTLS-SG/docs/extra/filters.html
+# pixel_scale, sky_brightness: http://www.cfht.hawaii.edu/Instruments/Imaging/Megacam/generalinformation.html
+# exp_time, fwhm: https://www.cfht.hawaii.edu/Science/CFHTLS/T0007/CFHTLS_T0007-TechnicalDocumentation.pdf
+# zeropoint: http://www1.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/community/CFHTLS-SG/docs/extra/filters.html
 CFHT:
   name: "CFHT"
-  pixel_scale: 0.185
+  pixel_scale: 0.187
   effective_area: 8.022
   mirror_diameter: 3.592
   airmass: 1.0 # TODO: double-check
```

Comment:
Should be 1.2, but I did not find the zeropoint airmass so we should also set that one to 1.2.

Reply:
Ok I changed it. Do you have a reference for this value ?

Reply:
https://www.cfht.hawaii.edu/Instruments/Imaging/Megacam/dietmegacam.html